internal/app/app_test.go

```diff
@@ -37,8 +36,8 @@ func TestNewPolicyAutomationApp(t *testing.T) {
 	if paApp.out == nil {
 		t.Fatalf("policyAutomationApp output is nil")
 	}
-	if !reflect.DeepEqual(paApp.out.w, io.Discard) {
-		t.Errorf("policyAutomationApp output is not io.Discard")
+	if paApp.collector == nil {
+		t.Fatalf("policyAutomationApp collector is nil")
```

Comment:
In addition I would check if the `ConsoleOutputCollector` is set (the default one).
This can be done with [Go type assertion](https://go.dev/ref/spec#Type_assertions), i.e.: 
```
	if _, ok := paApp.collector.(*outputs.ConsoleResultCollector); !ok {
		t.Fatalf("policyAutomationApp collector is not ConsoleResultCollector (default)")
	}
```